src/main/java/com/hms_networks/americas/sc/extensions/eventfile/EventFile.java

```diff
@@ -123,4 +140,50 @@ private static boolean checkEventFileLine(String line, String target) {
     }
     return currentToken.equals(target);
   }
+
+  /**
+   * Get the Export Block Descriptor request for Event File lines after the timestamp argument.
+   *
+   * @param timestampMilliseconds - the start time for request
+   * @return request string
+   */
+  private static String getEbdEventFileSinceAbsoluteTimeRequest(long timestampMilliseconds) {
+    Date TimeDateObj = new Date(timestampMilliseconds);
+    SimpleDateFormat sf = new SimpleDateFormat(FLEXY_EBD_EVENT_FILE_DATE_FORMAT);
+    return EBD_EVENT_LOG_ABSOLUTE_START_BASE + sf.format(TimeDateObj);
+  }
+
+  /**
+   * Read Event File via EBD using absolute start time, check for event Id
+   *
+   * @param eventId - ID of the target event
+   * @param absoluteStartTimeMilliseconds - epoch time in milliseconds as start time
+   * @return boolean - true/false found event ID in logs
+   * @throws EbdTimeoutException - for timeout of EBD request
+   * @throws IOException - for exception with input stream
+   */
+  public static boolean didEventOccurSinceAbsolute(
+      String eventId, long absoluteStartTimeMilliseconds) throws IOException, EbdTimeoutException {
+    Exporter ex =
+        HistoricalDataManager.executeEbdCall(
+            getEbdEventFileSinceAbsoluteTimeRequest(absoluteStartTimeMilliseconds));
+    String exporterFile = StringUtils.getStringFromInputStream(ex, EVENT_FILE_STRING_ENCODING);
+    ex.close();
+
+    if (exporterFile.length() > 0) {
+      final List eventFileLines = StringUtils.split(exporterFile, EVENT_FILE_LINE_DELIMITER);
+      for (int x = 1; x < eventFileLines.size(); x++) {
+        String line = (String) eventFileLines.get(x);
+        final List tokens = StringUtils.split(line, EVENT_FILE_LINE_TOKEN_DELIMITER);
```

Comment:
Is it possible that an event itself can contain the character `;`? If so, we should use `QuoteSafeStringTokenizer` here instead. An example of this can be found in the historical data manager: https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/blob/550a530a65d67cfa96c51355b29d612dcff9b4d8/src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataManager.java#LL408C5-L408C30

Reply:
I think the only circumstance it if a use does the following in basic
`logevent "blah;blah;"`
However, I will change to safe tokenizer to handle this case

Reply:
@alexjhawk  @oliver-walker-hms  testing complete.

Reply:
@alexjhawk  @oliver-walker-hms  testing complete.